sign/clin_sig/meta.yaml

```diff
@@ -0,0 +1,14 @@
+name: clin_sig
+type: "column"
+description: |
+  This spell visualizes the clinical significance via bootstrap pills
```

Comment:
Please avoid a bootstrap dependency in such a spell.

Reply:
Explain how the column should be formatted (what are the expected values, how are they separated).

Reply:
Mention that we talk about clinvar (https://www.ncbi.nlm.nih.gov/clinvar/) significance terms here.